bittensor/_dataloader/dataloader_impl.py

```diff
@@ -80,9 +81,9 @@ def requests_retry_session(
         session.mount('http://', adapter)
         session.mount('https://', adapter)
         return session
-    
+
     def retrieve_directory(self, dir_hash: str):
-        """Connects to Infura IPFS gateway and retrieves the directory of 
+        """Connects to Infura IPFS gateway and retrieves the directory of
```

Comment:
Should probably change that... we are using the pinata gateway right now

Reply:
Just control-f any reference to Infura, because that's all outdated now

Reply:
Agree. Lets remove all references to infura